src/js/startApp.js

```diff
@@ -1,17 +1,31 @@
-import { Station } from './Station';
-import { SubwayLine } from './SubwayLine';
-import { Subway } from './Subway';
-import { Route } from './Route';
+import { parseToSubway } from './Parser'
 
 let fileReader = new FileReader();
 let textInput = document.querySelector('#file-input');
-let subwayData = null;
+let currentSubway = null; 
+const localStorageKey = 'data';
```

Comment:
localStorageKey of what? such constants should contain information about what they are key for